src/azure-cli/azure/cli/command_modules/storage/commands.py

```diff
@@ -589,31 +600,39 @@ def get_custom_sdk(custom_module, client_factory, resource_type=ResourceType.DAT
         g.custom_command('snapshot', 'snapshot_share_rm', min_api='2020-08-01-preview', is_preview=True,
                          transform=transform_share_rm_output)
 
+    with self.command_group('storage share', command_type=share_client_sdk, is_preview=True,
```

Comment:
```suggestion
    with self.command_group('storage share', command_type=share_client_sdk,
```
It's already GA status no matter which SDK we use